starters/nextjs-starter-ts/components/smart-components/index.ts

```diff
@@ -26,6 +26,6 @@ export const serverSmartComponentMap = {
 export const clientSmartComponentMap: SmartComponentMap = {
   LEAD_CAPTURE: {
     ...serverSmartComponentMap.LEAD_CAPTURE,
-    reactComponent: LeadCapture,
+    reactComponent: LeadCapture as any,
```

Comment:
We need `LeadCapture` to be `() => JSX.Element`, but its type is derived from `forwardRef` wrapper which is different.
Without this change build works fine, but linting fails.
Tried updating type of `SmartComponentMap`, but its not possible since we are using `reactComponent` key as JSX element in the SDK.

Any suggestions?

Reply:
We should remove the ref forwarding here and in other smart components where it exists. It isn't used by the SDK.